src/openloco/company.h

```diff
@@ -15,11 +15,11 @@ namespace openloco
         constexpr company_id_t null = std::numeric_limits<company_id_t>::max();
     }
 
-    enum company_flags
+    namespace company_flags
     {
-        challenge_completed = (1 << 6),
-        challenge_failed = (1 << 7),
-        challenge_flag8 = (1 << 8),
+        constexpr uint32_t challenge_completed = (1 << 6); // 0x40
```

Comment:
I think we should leave this an `enum`. It's the more concise syntax of the two, and constexpr by definition.